ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -178,8 +179,22 @@ ReadButtons.propTypes = {
   showReads: PropTypes.func,
 }
 
+const optionReducer = trackType => (acc, { type, field, value }) => ({
+  ...acc,
+  ...(trackType === type) ? { [field]: value } : {},
```

Comment:
I think this would be more readable as 
```
trackType === type ? { ...acc, [field]: value } : acc
```
and I also think it needs a more descriptive name, maybe `addOptionsToTrackType`